ui/src/components/graph/utils.ts

```diff
@@ -182,7 +182,22 @@ const generateEdge = ({ obj, from, to }: GenerateEdgeProps): Edge => {
   };
 };
 
-export { generateEdge, generateNode, getLayoutedElements, getElements };
+const enum NodeTypes {
```

Comment:
>NodeTypes [](http://example.com/codeflow?start=11&length=9)

It's better name it FeatureType so non graph code can uses it too. And this should be moved to utils under ui root directory for same purpose

Reply:
Modified as required.